controllers/init/deploy/manifests/deploy.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: init-job
+spec:
```

Comment:
can use `spec.ttlSecondsAfterFinished` to set the automatic destruction time after successful automatic initialization.